setup.py

```diff
@@ -1,24 +1,36 @@
 #!/usr/bin/env python
+import os
 
-from setuptools import find_packages, setup
+from setuptools import setup, find_packages
+
+_PATH_ROOT = os.path.dirname(__file__)
+
+
+def _load_requirements(path_dir=_PATH_ROOT, comment_char="#"):
```

Comment:
This is a bit brittle, for example with empty lines or flags besides -r, although there isn't really a supported parser with modern pip:

https://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py

And it seems like reading requirements.txt in setup.py is kinda discouraged https://caremad.io/posts/2013/07/setup-vs-requirement/

Not sure if there would be a better way to go about it, any ideas?

Reply:
Yeah, I don't know better way...